src/microbe_stage/Microbe.Interior.cs

```diff
@@ -205,8 +205,8 @@ public void EmitToxin(Compound? agentType = null)
 
         var props = new AgentProperties(Species, agentType);
 
-        // Find the direction the microbe is facing
-        var direction = (LookAtPoint - Translation).Normalized();
+        // Find the direction the microbe is facing (actual rotation, not LookAtPoint)
+        var direction = new Vector3((float)-Math.Sin(Rotation.y), 0, (float)-Math.Cos(Rotation.y)).Normalized();
```

Comment:
Don't use angles unless absolutely required, use quaternions instead. Also this doesn't get the global rotation, so this'll be incorrect in microbe colonies.

I suggest using something like `GlobalTransform.basis.Quat()` and working from there.